### PR TITLE
Added module SSIDManager

### PR DESCRIPTION
**Pull request by noncenz**
This is a new module to allow easy switching between pools of SSIDs

### CODE REVIEW

File: SSIDManager/module.info [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "title": "SSID Manager",
+    "description": "Manage SSID Pools for PineAP",
+    "version": "1.0",
+    "author": "noncenz"
```
Comment (Foxtrot) [inline SSIDManager/module.info]: The module.info JSON requires an array of devices.
```
"author": "noncenz",
"devices": [
     "nano",
     "tetra"
],
```

If you want to have the module on both devices, for example.

File: SSIDManager/api/module.php [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                $this->downloadSSIDFile();
+                break;
+        }
+    }
+
+    private function SSIDDirectoryPath()
+    {
+        $path = '/pineapple/modules/SSIDManager/SSID_Files/';
+        if (!file_exists($path)) {
+            exec("mkdir ".$path);
```
[the remark this answers is not in the document]
Comment (noncenz) [reply SSIDManager/api/module.php]: Done!

File: SSIDManager/api/module.php [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    private function downloadSSIDFile()
+    {
+        $filename = $this->SSIDDirectoryPath().$this->request->file;
+        $this->response = array("download" => $this->downloadFile($filename));
+    }
+
+    private function deleteSSIDFile()
+    {
+        exec("rm -rf " . $this->SSIDDirectoryPath() ."'" . $this->request->file . "'");
```
[the remark this answers is not in the document]
Comment (noncenz) [reply SSIDManager/api/module.php]: Done!

File: SSIDManager/module.info (hunk 1 above)
Comment (noncenz) [reply SSIDManager/module.info]: All fixed, thanks for the help!

**Comment by Foxtrot** [comment]
Thank you!

**Pull request merged**